typescript/sdk/src/ism/EvmIsmModule.ts

```diff
@@ -536,12 +537,12 @@ export class EvmIsmModule extends HyperlaneModule<
     config: AggregationIsmConfig;
     logger: Logger;
   }): Promise<IAggregationIsm> {
-    const addresses: Address[] = await Promise.all(
-      config.modules.map(async (module) => {
-        const submodule = await this.deploy({ config: module });
-        return submodule.address;
-      }),
-    );
+    const addresses: Address[] = [];
+    // Needs to be deployed sequentially because Ethers will throw `Error: replacement fee too low`
```

Comment:
said differently - you can only send one transaction at a time. theoretically maybe the submitter or something could handle this queuing?